pkg/model/eviction_index.go

```diff
@@ -1,20 +1,26 @@
 package model
 
-import iotago "github.com/iotaledger/iota.go/v4"
+import (
+	iotago "github.com/iotaledger/iota.go/v4"
+)
 
 type EvictionIndex struct {
 	index *iotago.SlotIndex
 }
 
-func (e EvictionIndex) ShouldEvict(newIndex iotago.SlotIndex) bool {
+func NewEvictionIndex() *EvictionIndex {
```

Comment:
do we need an empty `New` here or should the user just call `new(model.EvictionIndex)` ?

Reply:
convention for almost all of our data types are they have a constructor. also we could add an optional initial index as a parameter to it